keyboards/otaku_split/rev0/readme.md

```diff
@@ -0,0 +1,15 @@
+# otaku_split
+
+![otaku_split](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [takashiski](https://github.com/yourusername)  
+Hardware Supported: The PCBs, controllers supported  
+Hardware Availability: links to where you can find this hardware
```

Comment:
Could you fill this out, if possible? 

Also, is this being sold, a group buy being ran, or just hosting the PCBs somewhere for people to get made? 

Depending, this board may be better in the "handwired", which is for personal/passion projects, as well as actually handwired boards.